ros_ws/launch/qlearning.launch

```diff
@@ -21,19 +22,5 @@
       <arg name="mode_override" value="2"/>
   </include>
 
-  <!-- Q-Learning related nodes: -->
-
-  <node
-    pkg="q_learning"
-    type="train.py"
-    name="q_learning_training"
-    output="screen" >
-  </node>
-
-  <node
-      pkg="simulation_tools"
-      type="crash_detector"
-      name="crash_detector"
-      output="screen" >
-  </node>
+  <include file="$(find q_learning)/launch/q_learning.launch"/>
```

Comment:
Why did you move these?

Reply:
I personally think it is better to have more modular launch files, I think it helps code readability. If you don't agree though or if it messes with your workflow I can revert it of course.